kolibri/core/apps.py

```diff
@@ -40,6 +42,11 @@ def ready(self):
             )
         )
         self.check_redis_settings()
+        # Do this to add an automapping from the Morango UUIDField to the UUIDFilter so that it automatically
+        # maps to this filter when using the UUIDField in a filter.
+        from morango.models import UUIDField
+
+        FilterSet.FILTER_DEFAULTS.update({UUIDField: {"filter_class": UUIDFilter}})
```

Comment:
I didn't know about the existence of `FILTER_DEFAULTS` nor I could find it in the documentation https://django-filter.readthedocs.io/ 
I only have found some references in the django-filter forums to underestand how it worked. Looking at the code makes sense, and also, if I understand it correctly the place in the code where you added it forces it to be active during the execution of the whole django request. is that interpretation correct?

Reply:
Yes - it does not appear to be documented, which is unfortunate, but it seemed to be the only way to set this global mapping from Field type to Filter class.

The other alternative would be to subclass FilterSet and then systematically use that for all our FilterSets (although, this would not work for automatically generated FilterSets using DRF's magic: https://www.django-rest-framework.org/api-guide/filtering/#djangofilterbackend as there doesn't seem to be a setting to set the default FilterSet class).

Setting it here seems to be early enough in the Django initialization sequence that it is now set for any derived class - because of the import path from morango, it needs to happen after Django app initialization, because of access to the models module.